### PR TITLE
Stylus to sass

### PR DESCRIPTION
**Pull request by KrystofM**

### CODE REVIEW

File: theme/styles/parts/shape.sass [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   * Mirrored views
+   */
+  &.view-180deg
+    transform: rotate(180deg)
+
+  &.view-90deg
+    transform: rotate(90deg)
+
+  &.view-270deg
+    transform: rotate(270deg)
```
Comment (ViliamKopecky) [inline theme/styles/parts/shape.sass]: I would add

```sass
&.view-flip-x
  transform: scaleX(-1)

&.view-flip-y
  transform: scaleY(-1)
```

@enzy also to mango-cli-example?

File: theme/styles/_global.sass [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/*! Default styles from your beloved manGoweb (http://mangoweb.cz) */
```
Comment (ViliamKopecky) [inline theme/styles/_global.sass]: Lets change it to https.

File: theme/styles/_global.sass [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+*, *:before, *:after
+	box-sizing: border-box
+
+html
+	font-size: $base-font-size
```
Comment (ViliamKopecky) [inline theme/styles/_global.sass]: might be better like this:

```sass
*, *:before, *:after
  box-sizing: inherit

html
  box-sizing: border-box
```

What do we think? @enzy @Onset

File: theme/styles/wp-admin.sass [hunk 4]
```diff
@@ -0,0 +1,2 @@
+
+@import 'wp-admin/*'
```
Comment (ViliamKopecky) [inline theme/styles/wp-admin.sass]: Add a new line at the end of the file

File: theme/styles/wp-editor.sass [hunk 5]
```diff
@@ -0,0 +1,2 @@
+
+@import 'wp-editor/*'
```
Comment (ViliamKopecky) [inline theme/styles/wp-editor.sass]: new line

File: theme/styles/_global.sass (hunk 3 above)
Comment (Onset) [reply theme/styles/_global.sass]: Looks good to me. 👍

**Comment by enzy** [comment]
I would rather switch our obsolete `_global.sass` with Bootstraps [Reboot](https://getbootstrap.com/docs/4.0/content/reboot/) in both codebases.

**Comment by ViliamKopecky** [comment]
@enzy thats what i was saying a year ago and you were against 😂

**Comment by enzy** [comment]
I know. Now it's in a beta and I already used it on a couple of projects. So it's a go from me ;)

**Pull request merged**